2017-2018/botCode/messages/Makefile

```diff
@@ -1,2 +1,4 @@
-protobuf: lightState.proto
+protobuf: lightState.proto pacmanState.proto
 	protoc -I=./ --python_out=./ ./lightState.proto
+	protoc -I=./ --python_out=./ ./pacmanState.proto
+	protoc -I=./ --python_out=./ ./pacmanCommand.proto
```

Comment:
Does protoc not support multiple input files at once?

Reply:
no idea